src/privatesend/privatesend-client.cpp

```diff
@@ -1626,8 +1622,10 @@ bool CPrivateSendClientSession::CreateDenominated(CAmount nBalanceToDenominate,
                          "CPrivateSendClientSession::CreateDenominated -- 1 - totalOutputs: %d, nOutputsTotal: %d, nOutputs: %d, nValueLeft: %f\n",
                          nOutputsTotal + nOutputs, nOutputsTotal, nOutputs, (float) nValueLeft / COIN);
 
+                if (nOutputs + nOutputsTotal >= MAX_PRIVATESEND_DENOM_OUTPUTS) break;
             }
             nOutputsTotal += nOutputs;
+            if (nOutputsTotal >= MAX_PRIVATESEND_DENOM_OUTPUTS) break;
```

Comment:
I don't see the point of adding these breaks, the 500 output check is primarily to ensure that the wallet doesn't become (overly) unresponsive, in actuality on develop it'll create txes up to 550 outputs, but I don't think that's a problem that needs to be fixed. Adding four breaks just doesn't seem worth it...

Reply:
agree partially, see 8b49159 kept them in "hard cap" cause there are no limits besides the balance otherwise